wordcloud/management/commands/generate_wordcloud.py

```diff
@@ -0,0 +1,37 @@
+import os
+
+from django.core.management.base import BaseCommand, CommandError
+from django.utils import simplejson
+from django.conf import settings
+
+from wordcloud.wordcloud import popular_words
+
+
+class Command(BaseCommand):
+    """Generate json file for wordcloud.
+
+    A single argument, a file path, can be passed in. This will
+    be appended to settings.WORDCLOUD_CACHE_DIR to produce the location
+    to store the json. If no argument is passed, it defaults to
+    'wordcloud.json'. If the file path is absolute it will just be used
+    as is.
+    """
+
+    help = 'Generate wordcloud data'
+    args = '<file>'
+
+    def handle(self, *args, **options):
+        if len(args) > 1:
+            raise CommandError
+
+        cache_dir = settings.WORDCLOUD_CACHE_DIR
+        try:
+            os.mkdir(cache_dir)
+        except OSError:
+            # Probably the directory exists already
```

Comment:
You can check if exc.errno == errno.EEXIST - this code's already `pombola/kenya/management/commands/iebc_api.py` so it might be worth moving that definition of mkdir_p somewhere common.